src/services/logger.js

```diff
@@ -1,7 +1,7 @@
-import bunyan from 'bunyan'
+import Bunyan from 'bunyan'
```

Comment:
capitalized for some reason?

Reply:
Any time when we do new constructor, we should be doing capitalize 
`[eslint] A constructor name should not start with a lowercase letter. (new-cap)`
 See: https://eslint.org/docs/rules/new-cap for more details